modules/order/src/Entity/OrderItem.php

```diff
@@ -108,6 +110,26 @@ public function getQuantity() {
   /**
    * {@inheritdoc}
    */
+  public function getItemsQuantity() {
```

Comment:
The (int) casting could be removed now but I don't the this was a reason for a quantity bug. It was quick and dirty solution to prevent displaying quantities like this in the  Shopping Cart Block:
`
8.25 items
`